ecf/scripts/plots/mesoscale/jevs_mesoscale_grid2obs_plots.ecf

```diff
@@ -50,7 +50,7 @@ if [ -n "%VHR:%" ]; then
 else
   export vhr=00
 fi
-export nproc=128
+export nproc=64
 export evs_run_mode="production"
 export USE_CFP=YES
```

Comment:
@malloryprow I made this change in my current PR #603 .  I guess it doesn't matter if we make this change twice.